src/loaders/mod_load.c

```diff
@@ -463,8 +493,14 @@ static int mod_load(struct module_data *m, HIO_HANDLE *f, const int start)
 	    break;
 	}
     }
+    #endif
 
     if (mod->chn == 0) {
+	#ifndef LIBXMP_CORE_PLAYER
```

Comment:
```suggestion
	#ifdef LIBXMP_CORE_PLAYER
```

Reply:
Thanks for catching it. Fixed and force-pushed.